src/Configurator.php

```diff
@@ -2,34 +2,135 @@
 
 namespace TomPHP\ConfigServiceProvider;
 
-interface Configurator
+use TomPHP\ConfigServiceProvider\FileReader\FileLocator;
+use TomPHP\ConfigServiceProvider\FileReader\ReaderFactory;
+use TomPHP\ConfigServiceProvider\Exception\NoMatchingFilesException;
+use TomPHP\ConfigServiceProvider\Exception\UnknownSettingException;
```

Comment:
@tomphp 

How about removing these imports and referencing the classes relative to the current namespace?